app/models/audience_segment.rb

```diff
@@ -0,0 +1,63 @@
+class AudienceSegment < ApplicationRecord
+  # enum does not like names that start with "not_"
+  enum type_of: {
+    manual: 0, # never matches anyone, used in test factory
```

Comment:
@benhalpern I did a quick spike on a cron- and ad-based refresh and changed the enum's default value from `testing` to `manual` — I think we can later expand this to allow setting a list of user-ids that _don't_ automatically refresh.

Reply:
Makes sense. I like it.